packages/desktop-client/src/components/sidebar/SidebarWithData.tsx

```diff
@@ -63,10 +68,11 @@ function EditableBudgetName({ prefs, savePrefs }) {
           }}
           defaultValue={prefs.budgetName}
           onEnter={async e => {
-            const newBudgetName = e.target.value;
+            const inputEl = e.target as HTMLInputElement;
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏why do we need this manual type coercion? `e` should already be `KeyboardEvent<HTMLInputElement>`.

Reply:
Ok, lets keep it as-is. Maybe we can fix it somehow later, but either way it's a nitpick.